manim/mobject/mobject.py

```diff
@@ -2337,9 +2337,9 @@ def shuffle_submobjects(self, *args, **kwargs):
         Examples
         --------
 
-        .. manim:: SuffleSumobjectsExample
+        .. manim:: ShuffleSumobjectsExample
```

Comment:
You caught one spelling mistake but not the other 😛
```suggestion
        .. manim:: ShuffleSubmobjectsExample
```

Reply:
Upsi :D